diff -aurN gtkperf.orig/data/gtkperf.desktop gtkperf/data/gtkperf.desktop
--- gtkperf.orig/data/gtkperf.desktop	2009-12-31 13:47:35.000000000 +0100
+++ gtkperf/data/gtkperf.desktop	2009-12-31 13:50:20.000000000 +0100
@@ -1,8 +1,9 @@
-[Desktop Entry] 
-Encoding=UTF-8 
-Version=1.0 
-Type=Application 
+[Desktop Entry]
+Type=Application
 Name=GtkPerf
-Exec=/usr/bin/gtkperf
-X-Osso-Service=com.nomovok.gtkperf
-Icon=qgn_list_gene_default_app
+Exec=gtkperf
+Icon=gtkperf
+Comment=GTK+ performance tester
+Terminal=false
+Categories=GTK;Utility;
+
